JSDemos/Demos/RangeSelector/UseRangeSelectionForCalculation/React/App.js

```diff
@@ -8,6 +8,8 @@ const startValue = new Date(2011, 0, 1);
 const endValue = new Date(2011, 11, 31);
 const behaviorModes = ['onHandleMove', 'onHandleRelease'];
 
+const behaviorLabel = { 'aria-label': 'Behavior' };
```

Comment:
```suggestion
const valueChangeModeLabel = { 'aria-label': 'Value Change Mode' };
```
Please, update also the appropriate demo code and fix it in all frameworks